OKTA-809892 Add API service integration in OIN Manager docs (#5279)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

### Scope selection

> Specify the resources and actions that your API service app requires to do its job. A scope corresponds to a resource that you want to access in the Okta API (users, logs, and so on) and a level of access (read or manage). Use the least-privilege principle and authorize the fewest scopes needed for your app to function correctly. See the list of available Okta [OAuth 2.0 Scopes](https://developer.okta.com/docs/api/oauth2/).

Reviewer: ```suggestion
Specify the resources and actions that are required in order for your API service app to work. A scope corresponds to a resource that you want to access in the Okta API (users, logs, and so on) and a level of access (read or manage). Use the least-privilege principle and authorize the fewest scopes needed for your app to function correctly. See the list of available Okta [OAuth 2.0 Scopes](https://developer.okta.com/docs/api/oauth2/).
```
